### ISSUE TITLE
wrong model weights saved

### ISSUE THREAD

**Issue by sharminpathan** [sassoftware/python-dlpy#20]
Hi,
i tried doing k-fold cross validation while training to fine tune the weights. on the 1st fold i got an error of 17% and at the last fold i got the error down to 4%. when i save the model weights and try to load them, i notice that i get an error rate associated with the first fold (17%). is there something am doing wrong?

Here's the code:
for i in range(18): 
    
        print("Training pass "+str(i))

        train, valid = sampling(conn)
        
        model.fit(data=train, inputs='Survey_Verbatim', texts='Survey_Verbatim', target='Section', nominals='Section', valid_table=valid,
                text_parms=TextParms(init_input_embeddings='glove'), record_seed=909455678,
                mini_batch_size=1, max_epochs=n_epochs, lr=0.01, log_level=2)
        
        n_epochs = 10

        model.save_to_table('/bigdisk/lax/shpath/')

**Comment by ethem-kinginthenorth** [comment]
I am assuming this is the error in the training set. Are you saving the folds or creating them on-the-fly?

**Comment by sharminpathan** [comment]
i evaluated the model in the same session on a separate held out test set, the error is 4%.
then i save the model weights, load them in a different session and evaluate it against the same test set, now the error is 17%

**Comment by ethem-kinginthenorth** [comment]
Ok, I am looking into the issue.

**Comment by ethem-kinginthenorth** [comment]
the issue was using different test tables. so this is not a bug.
